Docs/reference/content/reference/driver/crud/client_side_encryption.md

```diff
@@ -0,0 +1,197 @@
+
++++
+date = "2019-09-30T20:38:-04:00"
+title = "Client-Side Field Level Encryption"
+[menu.main]
+  parent = "Reference Reading and Writing"
+  identifier = "Client-Side Field Level Encryption"
+  weight = 40
+  pre = "<i class='fa fa-lock'></i>"
++++
+
+# Client-Side Field Level Encryption
+
+New in MongoDB 4.2, client-side field level encryption allows administrators and
+developers to encrypt specific data fields in addition to other MongoDB
+encryption features.
+
+With client-side field level encryption, developers can encrypt fields
+client-side without any server-side configuration or directives. Client-side
+field level encryption supports workloads where applications must guarantee that
+unauthorized parties, including server administrators, cannot read the encrypted
+data.
+
+{{% note class="important" %}} 
+Support for client-side field level encryption is in beta, and the beta only
+supports Windows. Backwards-breaking changes may be made before the
+final release (which will support MacOS and Linux).
+{{% /note %}}
+
+## libmongocrypt
+
+`libmongocrypt` is a required component for client-side field level encryption
+and is included in the `MongoDB.Libmongocrypt` package.
```

Comment:
Is there still some chance for confusion here?

"required component" and "included in ... package" make it sound like the developer hast to DO something about it.

But I'm going to approve anyway. Optional if you want to do any rewording or not.

Reply:
If the intent is simply to inform the developer that some new files exist perhaps that can be expressed more clearly. Something like:

Client-side field level encryption is implemented in two new files: `MongoDB.Libmongocrypt.dll` and `libmongocrypt.dll`. These files are part of the new `MongoDB.Libmongocrypt` Nuget package and are automatically included in your application.

Reply:
Discussed offline!